core/container/dockercontroller/dockercontroller.go

```diff
@@ -247,7 +248,7 @@ const (
 	TLSClientRootCertFile string = "/etc/hyperledger/fabric/peer.crt"
 )
 
-func (vm *DockerVM) GetEnv(ccid string, tlsConfig *ccintf.TLSConfig) []string {
+func (vm *DockerVM) GetEnv(ccid string, tlsConfig *ccintf.TLSConfig, mspid string) []string {
```

Comment:
Haven't you already wired the MSPID to the DockereVM, no need to pass it into the function, can pull it right off of `vm`

Reply:
good catch thanks!